spec/factories/distributions.rb

```diff
@@ -19,6 +20,7 @@
     partner
     organization { Organization.try(:first) || create(:organization) }
     issued_at { nil }
+    state { 0 }
```

Comment:
Not a blocker and no-need to fix: You can also just use the symbol of the state, here! 

```ruby
state { :started }
```